projects/demo/src/modules/app/home/examples/main-standalone-optional.md

```diff
@@ -1,8 +1,18 @@
 ```ts
-import {provideAnimations} from '@angular/platform-browser/animations';
-import {TuiAlertModule, TuiDialogModule, TuiRootModule} from '@taiga-ui/core';
+import {BrowserAnimationsModule} from '@angular/platform-browser/animations';
+import {TuiAlertModule, TuiDialogModule, TuiRootModule, TuiPushModule} from '@taiga-ui/core';
 
 bootstrapApplication(AppComponent, {
-  providers: [importProvidersFrom(TuiRootModule, TuiDialogModule, TuiAlertModule), provideAnimations()],
+  providers: [
+    importProvidersFrom(
+      BrowserAnimationsModule,
+      EventPluginsModule,
```

Comment:
You do not need it, it's under the hood of root.

Reply:
fixed